python/cugraph/layout/force_atlas2_wrapper.pyx

```diff
@@ -94,8 +94,8 @@ def force_atlas2(input_graph,
     # We keep np.float32 as results for both cases
     pos = cuda.device_array(
             (num_verts, 2),
-            order="F",
-            dtype=np.float32)
+            order="C",
+            dtype=np.int32)
```

Comment:
does this change the comment 4 lines above this?

Reply:
Will revert this change, forgot to clean it